compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -395,6 +395,8 @@
     MATCH_STMT_UNREACHABLE_PATTERN("match.stmt.unreachable.pattern"),
     MATCH_STMT_UNMATCHED_PATTERN("match.stmt.unmatched.pattern"),
     MATCH_STMT_PATTERN_ALWAYS_MATCHES("match.stmt.pattern.always.matches"),
+    MATCH_STMT_UNREACHABLE_PATTERN_AVAILABLE("match.stmt.unreachable.pattern.available"),
```

Comment:
shall we remove the last part `.available`

Reply:
changed